nixos/tests/all-tests.nix

```diff
@@ -35,6 +35,7 @@ in
   borgbackup = handleTest ./borgbackup.nix {};
   buildbot = handleTest ./buildbot.nix {};
   cadvisor = handleTestOn ["x86_64-linux"] ./cadvisor.nix {};
+  cassandra = handleTestOn ./cassandra.nix {};
```

Comment:
Do you want `handleTest` or `handleTestOn` [ "..." ]?

Reply:
Ooops, thanks!